include/js.h

```diff
@@ -1131,6 +1131,42 @@ js_get_array_length(js_env_t *env, js_value_t *value, uint32_t *result) {
   return js_convert_from_status(status);
 }
 
+static inline int
+js_get_array_elements(js_env_t *env, js_value_t *array, js_value_t **elements, size_t len, size_t offset, uint32_t *result) {
+  uint32_t array_len;
+
+  napi_status status = napi_get_array_length(env, array, &array_len);
+
+  if (status == napi_ok) {
```

Comment:
nit question: aren't flat conditionals/early-return a bit easier to read or are there side-effects of doing that?

Reply:
the logic was watertight either way, curiosity got the better me, neat!